### PR TITLE
Use standard "translate" marker in HTML, fix PO template build

### PR DESCRIPTION
**Pull request by martinpitt**
Cockpit's test-static-code complains about `translatable`, so let's use
the correct attribute to avoid spreading it further.

### CODE REVIEW

**Comment by martinpitt** [comment]
Note that our integration tests check that the title actually gets translated.

File: src/index.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
 -->
 <html lang="en">
 <head>
-    <title translatable="yes">Cockpit Starter Kit</title>
+    <title translate="yes">Cockpit Starter Kit</title>
```
Comment (mvollmer) [inline src/index.html:20]: Isn't just `<title translate>` enough? That would be best to show as an example, if correct.

**Review by mvollmer — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (martinpitt) [reply src/index.html:20]: I verified that it actually is, so I changed it accordingly.

**Comment by martinpitt** [comment]
I also noticed that building the pot is totally broken:
```
❱❱❱ make po/starter-kit.pot
po/html2po -f po/POTFILES.html.in -o po/starter-kit.html.pot
po/manifest2po src/manifest.json -o po/starter-kit.manifest.pot
Option "--directory" requires 1 arguments, but 0 were provided
USAGE: node manifest2po [OPTION1] [OPTION2]... arg1 arg2...
The following options are supported:
  -d, --directory <ARG1> 	Base directory for input files
  -o, --output <ARG1>    	Output file
  -f, --from <ARG1>      	File containing list of input files
Option "--directory" requires 1 arguments, but 0 were provided
USAGE: node html2po [OPTION1] [OPTION2]... arg1 arg2...
The following options are supported:
  -d, --directory <ARG1> 	Base directory for input files
  -o, --output <ARG1>    	Output file
  -f, --from <ARG1>      	File containing list of input files
make: *** [Makefile:45: po/starter-kit.manifest.pot] Error 1
make: *** Waiting for unfinished jobs....
make: *** [Makefile:42: po/starter-kit.html.pot] Error 1
```

Fixed as well, and added to travis.

File: Makefile [hunk 2]
```diff
@@ -39,10 +39,10 @@ po/POTFILES.html.in:
 	find src -name '*.html' > $@
 
 po/$(PACKAGE_NAME).html.pot: po/POTFILES.html.in
-	po/html2po -f $^ -o $@
+	po/html2po -d . -f $^ -o $@
 
-po/$(PACKAGE_NAME).manifest.pot:
-	po/manifest2po src/manifest.json -o $@
+po/$(PACKAGE_NAME).manifest.pot: src/manifest.json
+	po/manifest2po -d . -f $^ -o $@
```
Comment (martinpitt) [inline Makefile:45]: Meh, this isn't right either. Why are these tools to utterly complicated??

**Comment by martinpitt** [comment]
OK, now the generated pot file looks correct.

**Review by mvollmer — APPROVED** [review]

**Pull request merged**